test/end-to-end/company.js

```diff
@@ -162,6 +162,19 @@ describe("Company application endpoint", () => {
                     });
                 });
 
+                test("should return an error when the image size is greater than the max size", async () => {
+                    const res = await test_agent
+                        .post("/company/application/finish")
+                        .attach("logo", "test/data/logo-niaefeup-10mb.png")
+                        .expect(HTTPStatus.UNPROCESSABLE_ENTITY);
```

Comment:
Is it fine to include a 10MB+ image in the repos just for testing reasons? If not, how should we test this?

Reply:
Ideally, we should mock the api that multer uses to read the file, and make it return a fake file with a fake size of 10+MB, but this is easier, and it won't hurt the actual application performance, so I'm fine with it.